imports/plugins/included/discount-rates/server/methods/methods.app-test.js

```diff
@@ -42,7 +41,6 @@ describe("discount rate methods", function () {
       expect(discountInsertSpy).to.have.been.called;
 
       const discountCount = Discounts.find(discountId).count();
-      Meteor._sleepForMs(500);
```

Comment:
I'm getting test failures re-introduced by removing these sleeps.

Reply:
Is it okay if I try to fix the code instead? Needing to sleep for tests is a tenuous fix and just ignores the fact that failures due to timing usually mean poorly written code.

Reply:
For sure. I would love to get these really fixed. Just pointing out that the tests are failing intermittently now.